### PR TITLE
Use new table in user and user groups in project edit

### PR DESCRIPTION
**Pull request by AdityaKhatri**

### CODE REVIEW

File: src/views/ProjectEdit/Users/UserGroupList/index.tsx [hunk 1]
```diff
@@ -37,15 +37,65 @@ import {
...
+    }, [itemKey, onEditClick]);
+
+    const handleDeleteUserGroupClick = useCallback(() => {
+        onDeleteClick(itemKey);
+    }, [itemKey, onDeleteClick]);
+
+    return (
+        <div className={_cs(styles.actionCell, className)}>
+            <QuickActionButton
+                className={styles.button}
```
Comment (tnagorra) [inline src/views/ProjectEdit/Users/UserGroupList/index.tsx]: Please add title

File: src/views/ProjectEdit/Users/UserList/index.tsx [hunk 2]
```diff
@@ -1,50 +1,114 @@
...
+                onConfirm={handleDeleteMembershipClick}
+                message={_ts('projectEdit', 'removeUserConfirmation')}
+                showConfirmationInitially={false}
+                disabled={disabled}
+            >
+                <IoTrash />
+            </QuickActionConfirmButton>
+        </div>
+    );
+}
```
Comment (tnagorra) [inline src/views/ProjectEdit/Users/UserList/index.tsx]: Why was this not reused?

**Review by tnagorra — CHANGES_REQUESTED** [review]
Need to reuse ActionCell

File: src/views/ProjectEdit/Users/UserList/index.tsx (hunk 2 above)
Comment (AdityaKhatri) [reply src/views/ProjectEdit/Users/UserList/index.tsx]: too tired?

Comment (AdityaKhatri) [reply src/views/ProjectEdit/Users/UserList/index.tsx]: done :crying_cat_face:

**Review by tnagorra — APPROVED** [review]

**Pull request merged**